.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**
+
+## Description
+
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+Fixes # (issue)
+
+
+## New/Edited policies (Delete if not relevant)
+
+### Description
+*Include a description of what makes it a violation and any relevant external links.*
+
+### Fix
+*How do I fix the issue in code and/or in runtime?*
+
+### Category
+Kubernetes, IAM, Networking, Storage, Compute, Public, Secrets, Supply Chain, Monitoring
+
+### Severity
+LOW, MEDIUM, HIGH, CRITICAL
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my feature, policy, or fix is effective and works
+- [ ] New and existing pytest tests pass locally with my changes
```

Comment:
```suggestion
- [ ] New and existing tests pass locally with my changes
```
no need to especially name `pytest` because it is just a test runner 🙂

Reply:
Done!